javascript/javascript3/week1/Find-a-cool-api.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Find-a-cool-api</title>
+</head>
+<body>
+    <p>
+    List the cat breeds API : <a href="https://api.thecatapi.com/v1/breeds"> https://api.thecatapi.com/v1/breeds</a><br>
+    Method:Get <br>
+    returns: cat breeds as Json file  as array of objects<br>
+    Example format of response:  <br>
+    <br>  [ 
```

Comment:
To define an object, you don't need to use a line break element! It can be like this:
```
const arr= [
{},
{},
...
]
```